pkgs/development/libraries/science/math/libtorch/binary-hashes.nix

```diff
@@ -16,4 +16,21 @@ version : builtins.getAttr version {
       hash = "sha256-jPylK4j0V8SEQ8cZU+O22P7kQ28wanIB0GkBzRGyTj8=";
     };
   };
+  "1.12.1" = {
+    x86_64-darwin-cpu = {
+      name = "libtorch-macos-1.12.1.zip";
+      url = "https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.12.1.zip";
+      hash = "sha256-HSisxHs466c6XwvZEbkV/1kVNBzJOy3uVw9Bh497Vk8=";
+    };
+    x86_64-linux-cpu = {
+      name = "libtorch-cxx11-abi-shared-with-deps-1.12.1-cpu.zip";
+      url = "https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.12.1%2Bcpu.zip";
+      hash = "sha256-bHhC0WTli9vDJv56TaT6iA/d8im9zRkK1TlBuqkh4Wg=";
+    };
+    x86_64-linux-cuda = {
+      name = "libtorch-cxx11-abi-shared-with-deps-1.12.1-cu113.zip";
+      url = "https://download.pytorch.org/libtorch/cu113/libtorch-cxx11-abi-shared-with-deps-1.12.1%2Bcu113.zip";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/torch/binary-hashes.nix#L12
![image](https://user-images.githubusercontent.com/2469618/189809994-e2094aa1-f4dc-4b6b-94cd-d6819fd08d24.png)

We may prefer cu116 for the latest device support.

Reply:
Yes, you're right. I fixed this and hashes. Can i ask for review?